### PR TITLE
I1464 update meteor

### PR DESCRIPTION
**Pull request by Fallen1988**
meteor updated to v1.8

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,5 +61,8 @@
     "react-test-renderer": "^16.4.1",
     "sift": "^6.0.0",
     "styled-components": "^3.1.5"
+  },
+  "dependencies": {
+    "plio-util": "https://github.com/Pliohub/plio-util.git#devel"
```
Comment (Dartv) [inline package.json]: Actually, we need `plio-util` as **dev** dependency

**Comment by Dartv** [comment]
@Fallen1988 did you update babel runtime? https://github.com/meteor/meteor/blob/devel/History.md

**Comment by Fallen1988** [comment]
no. Do I need to update it ?

**Comment by Dartv** [comment]
Yes, read `Migration steps` from the link

**Pull request merged**